### ISSUE TITLE
Feature Request - add support for Visual Studio 2022 ARM

### ISSUE THREAD

**Issue by RonFieldsUBH** [ErikEJ/EFCorePowerTools#1659]
Would it be possible to add support for the ARM processor M1 etc.? I know this may be a daunting task but my particular setup is running Windows 11 ARM in Parallels on a new Mac Studio.
 
1/18/2023 11:01:30 AM - 	Locale             : en-US
1/18/2023 11:01:30 AM - 	MoreInfoURL        : https://github.com/ErikEJ/EFCorePowerTools/wiki/
1/18/2023 11:01:30 AM - 	InstalledByMSI     : False
1/18/2023 11:01:30 AM - 	SupportedFrameworkVersionRange : [4.8,)
1/18/2023 11:01:30 AM - 
1/18/2023 11:01:30 AM - 	SignatureState     : Unsigned
1/18/2023 11:01:30 AM - 	Supported Products : 
1/18/2023 11:01:30 AM - 		Microsoft.VisualStudio.Community
1/18/2023 11:01:30 AM - 			Version : [17.0,18.0)
1/18/2023 11:01:30 AM - 			ProductArchitecture : amd64
1/18/2023 11:01:30 AM - 
1/18/2023 11:01:30 AM - 	References         : 
1/18/2023 11:01:30 AM - 	Prerequisites      : 
1/18/2023 11:01:30 AM - 		-------------------------------------------------------
1/18/2023 11:01:30 AM - 		Identifier   : Microsoft.VisualStudio.Component.CoreEditor
1/18/2023 11:01:30 AM - 		Name         : Visual Studio core editor
1/18/2023 11:01:30 AM - 		Version      : [17.0,)
1/18/2023 11:01:30 AM - 
1/18/2023 11:01:30 AM - 		-------------------------------------------------------
1/18/2023 11:01:30 AM - 		Identifier   : Microsoft.VisualStudio.Component.GraphDocument
1/18/2023 11:01:30 AM - 		Name         : DGML editor
1/18/2023 11:01:30 AM - 		Version      : [17.0,)
1/18/2023 11:01:30 AM - 
1/18/2023 11:01:30 AM - Signature Details...
1/18/2023 11:01:30 AM - 	Extension is not signed.
1/18/2023 11:01:30 AM - 
1/18/2023 11:01:30 AM - Searching for applicable products...
1/18/2023 11:01:30 AM - Found installed product - Visual Studio Professional 2022
1/18/2023 11:01:30 AM - Found installed product - Global Location
1/18/2023 11:01:31 AM - VSIXInstaller.NoApplicableSKUsException: This extension is not installable on any currently installed products.
   at VSIXInstaller.ExtensionService.GetInstallableDataImpl(IInstallableExtension extension, String extensionPackParentName, Boolean isRepairSupported, IStateData stateData, IEnumerable`1& skuData)
   at VSIXInstaller.ExtensionService.GetInstallableData(String vsixPath, String extensionPackParentName, Boolean isRepairSupported, IStateData stateData, IEnumerable`1& skuData)
   at VSIXInstaller.ExtensionPackService.IsExtensionPack(IStateData stateData, Boolean isRepairSupported)
   at VSIXInstaller.ExtensionPackService.ExpandExtensionPackToInstall(IStateData stateData, Boolean isRepairSupported)
   at VSIXInstaller.App.Initialize(Boolean isRepairSupported)
   at VSIXInstaller.App.Initialize()
   at System.Threading.Tasks.Task`1.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()

**Comment by ErikEJ** [comment]
It looks like it is possible yes, but involves building a seperate extension (.vsix file) :-( 

https://devblogs.microsoft.com/visualstudio/now-introducing-arm64-support-for-vs-extensions/

**Comment by ErikEJ** [comment]
I implemented a fix for this in the [latest daily build](https://www.vsixgallery.com/extension/f4c4712c-ceae-4803-8e52-0e2049d5de9f), would be grateful if you could try it out.

**Comment by RonFieldsUBH** [comment]
I was able to install the 2.5.1271 daily build. The installation went good and I was able to establish a new database connection. When I click on Reverse Engineer the following error was available in the extension output window. Looks like the  efreveng60.dll is the culprit now.

System.InvalidOperationException: Table list error: 
Unhandled exception. System.BadImageFormatException: Could not load file or assembly 'C:\Users\rfields\AppData\Local\Temp\efreveng6.2.5.1271\efreveng60.dll'. An attempt was made to load a program with an incorrect format.
File name: 'C:\Users\rfields\AppData\Local\Temp\efreveng6.2.5.1271\efreveng60.dll'

   at List<TableModel> EFCorePowerTools.Handlers.ReverseEngineer.ResultDeserializer.BuildTableResult(string output)
   at async Task<List<TableModel>> EFCorePowerTools.Handlers.ReverseEngineer.EfRevEngLauncher.GetTablesInternalAsync(string arguments)
   at async Task<List<TableModel>> EFCorePowerTools.Handlers.ReverseEngineer.EfRevEngLauncher.GetTablesAsync(string connectionString, DatabaseType databaseType, SchemaInfo[] schemas, bool mergeDacpacs)
   at async Task<List<TableModel>> EFCorePowerTools.Handlers.ReverseEngineer.TableListBuilder.GetTableDefinitionsAsync(CodeGenerationMode codeGenerationMode)
   at async Task<List<TableModel>> EFCorePowerTools.Handlers.ReverseEngineer.ReverseEngineerHandler.GetTablesAsync(DatabaseConnectionModel dbInfo, CodeGenerationMode codeGenerationMode, SchemaInfo[] schemas)
   at async Task<bool> EFCorePowerTools.Handlers.ReverseEngineer.ReverseEngineerHandler.LoadDataBaseObjectsAsync(ReverseEngineerOptions options, DatabaseConnectionModel dbInfo, Tuple<List<Schema>, string> namingOptionsAndPath)
   at async Task EFCorePowerTools.Handlers.ReverseEngineer.ReverseEngineerHandler.ReverseEngineerCodeFirstAsync(Project project, string optionsPath, bool onlyGenerate, bool fromSqlProj)
